tests/e2e/go.mod

```diff
@@ -144,3 +144,11 @@ require (
 	nhooyr.io/websocket v1.8.6 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace (
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.12-0.20221116140330-9c145c827001
+	github.com/cosmos/ibc-go/v3 => github.com/informalsystems/ibc-go/v3 v3.4.1-0.20221202165607-3dc5ba251371
```

Comment:
Are we okay with using our branch of ibc-go?

Reply:
That's a really good question. IIRC our changes are just testing related, so it's possible that we could use mainline 3.4 here without breaking anything in the actual code

Reply:
Yeah just reverted

Reply:
@jtremback @mpoke We actually need to keep the dependency for [informalsystems/ibc-go](https://github.com/informalsystems/ibc-go/commit/3dc5ba251371c39d2358ab5578484485cf15cc06) as otherwise GaiaApp will be incompatible and unable to implement [`e2eutil.ProviderApp`](https://github.com/cosmos/interchain-security/blob/main/tests/e2e/instance_test.go#L60). This is because ICS's e2e `ProviderApp` must implement the following which is not available if we use  IBC `v3.4.1`
```go
import ibcclienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"

func (appKeepers *AppKeepers) GetStakingKeeper() ibcclienttypes.StakingKeeper {  
  return appKeepers.StakingKeeper  
}
```

Reply:
Does our fork of ibc-go change anything besides the testing framework?

Reply:
Will have a concrete update by Monday, discussed a possible solution for the release [here](https://github.com/cosmos/interchain-security/issues/585#issuecomment-1355153358).